src/main/resources/story/api/Pagination.story

```diff
@@ -0,0 +1,41 @@
+Narrative:
+This story is designed to validate REST web-service with 'vividus-plugin-rest-api'
+VIVIDUS features:
+line 20: compare actual value from response with global variable stored in property file
+line 26: usage of condition step. If number of palette entities more than 1 - sub-steps placed in table will be executed
+line 37: example table with dynamic values calculation, based on actual service response in the first scenario
+
+Meta:
+    @api
+    @feature 13
```

Comment:
`13` should be replaced with some name, this tag is used to group scenarios/stories within report